test/clienthellotest.c

```diff
@@ -23,7 +23,7 @@
 
 #define CLIENT_VERSION_LEN      2
 
-#define TOTAL_NUM_TESTS                         4
+#define TOTAL_NUM_TESTS                         3
```

Comment:
I somehow would have thought that tests would be added, not removed as part of this new feature, no?

Reply:
This comes from one of the commits I added. See my justification in the commit description for "Drop the session and PSK test from the clienthellotest". Basically this is a low value test (already covered by other test cases) which is very brittle.